lib/game.rb

```diff
@@ -28,7 +30,7 @@ def launch_game
           post_player_position(player_selection_index)
           switch_player
         else
-          puts 'Invalid Selection, Please selection 1 to 9'
+          puts '**INVALID** Please select a number between 1 and 9.'
```

Comment:
Something else to think about for the future/ refactoring:
@breadoliveoilsalt @twopoint718  
System outputs (prints/puts) might be better off in their own "console_output" class with methods for outputting.
This would clean up code a lot and separate logic further

Reply:
feature needs to be developed in a future story is time permits

Reply:
console class has been created and puts statements have been removed from the Game class main methods